department-of-veterans-affairs/caseflow
# add aria attributes to table header to reduce verbosity

Resolves #12094

### Description
Unable to exactly duplicate Issue #12094 using VoiceOver in Chrome, but the following may resolve it:
1. add aria-label for header column's sortIcon
2. add blank aria-label to prevent repeated column name

### Acceptance Criteria
- [ ] "filter by" is only read by VoiceOver when the filter icon is selected

### Testing Plan
1. Case details for any judge review task.
2. Enable screen reader (See instructions in #12095).
3. Navigate VoiceOver (VO) to each table header cell, noting that VO only reads the selection.
